modules/restoring-red-hat-quay.adoc

```diff
@@ -0,0 +1,290 @@
+[[restoring-up-red-hat-quay]]
+== Restoring Red Hat Quay
+
+This procedure is used to restore {productname} when the Red Hat Quay Operator manages the database. It should be performed after a backup of your Quay registry has been performed.
+
+
+.Prerequisites
+
+* {productname} is deployed on OpenShift Container Platform using the Quay Operator.
+* Your {productname} database has been backed up.
+
+.Procedure
+
+
+. Restore the backed up Quay configuration and the randomly generated keys:
++
+[source,terminal]
+----
+$ oc create -f ./config-bundle.yaml
+----
++
+[source,terminal]
+----
+$ oc create -f ./managed-secret-keys.yaml
+----
+
+. Restore the QuayRegistry custom resource:
++
+[source,terminal]
+----
+$ oc create -f ./quay-registry.yaml
+----
```

Comment:
line 19, 24 and 31 may be hit  one problem.  
If the resource names have been used in the cluster which is used to resotre quay, we need to delete the exisiting resource first.  Such as 

```
$ oc create -f ./config-bundle.yaml
Error from server (AlreadyExists): error when creating "./config-bundle.yaml": secrets "config-bundle-secret" already exists

$ oc delete Secret config-bundle-secret -n quay-enterprise
secret "config-bundle-secret" deleted

$ oc create -f ./config-bundle.yaml
```